Sources/Conformance/main.swift

```diff
@@ -54,7 +54,7 @@ func buildResponse<Bytes: SwiftProtobufContiguousBytes>(serializedData: Bytes) -
 
     let request: Conformance_ConformanceRequest
     do {
-        request = try Conformance_ConformanceRequest(contiguousBytes: serializedData)
+        request = try Conformance_ConformanceRequest(serializedBytes: serializedData)
```

Comment:
Just want to double check here with @thomasvl. Since we talked about keeping the breaking changes limited this is now quite a wide reaching one. Even though we still support `Data` based `init`s we changed the name of the parameter. I am thinking if we should still provide a `convenience init` in `SwiftProtobufCompat` that maintains the old naming with a deprecation so that adopters won't break on this right away?

Reply:
Part of me wishes we didn't change the tests/conformance tests/fuzz tests, so we can see that the foundation shims were all working (then clean them up in a follow up commit/pr).

Reply:
Agreed, the previous PR did some changes which we maybe should look into again.